feat: Add multiple collaborators to a ticket

Akashsah2003 opened the pull request:
### Description

*Please include a short description of the suggested change and the reasoning behind the approach you have chosen.*
This PR aims to add a new feature which is adding multiple collaborators to a ticket.
- Now we can add multiple collaborators in a ticket.
- These collaborators can track these tickets through their dashboard, calendar and project page.

### Link to ticket

*Please add a link to the GitHub issue being addressed by this change.*
Closes #1099 

### Type

- [ ] Fix
- [x] Feature
- [ ] Cleanup 

### Screenshot of the result
<img width="1850" height="1053" alt="Screenshot from 2025-11-01 17-25-42" src="https://github.com/user-attachments/assets/4bed5dfd-522a-4340-b496-eb0937749dc9" />
<img width="1850" height="1053" alt="Screenshot from 2025-11-01 17-27-27" src="https://github.com/user-attachments/assets/7ec1a017-347e-4cd5-9a28-c581a65c6ce2" />


*If your change affects the user interface, you should include a screenshot of the result with the pull request.*

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/Leantime/leantime?pullRequest=3157) <br/>All committers have signed the CLA.

marcelfolaron reviewed (approved):
Thank you! This is great!

Akashsah2003 commented:
> Can you take a look at the acceptance tests and make sure they pass? On your local machine you can run
> 
> `make run acceptance-test`

I have corrected the error. The issue was I corrected column name to `entityA` which was `enitityA` before in `zp_entity_relationship` but this change was not being correctly applied in fresh install. Now the acceptance tests are passing.
